pkg/helpers/image/test/util.go

```diff
@@ -184,10 +188,13 @@ func PodSpec(containerImages ...string) corev1.PodSpec {
 }
 
 // StreamList turns the given streams into StreamList.
-func StreamList(streams ...imagev1.ImageStream) imagev1.ImageStreamList {
-	return imagev1.ImageStreamList{
-		Items: streams,
+func StreamList(streams ...imagev1.ImageStream) map[string]*imagev1.ImageStream {
```

Comment:
Rename this function? It does not "turn the given streams into StreamList" (anylonger) :-D

Reply:
done